tmt/steps/prepare/__init__.py

```diff
@@ -69,11 +69,7 @@ def go(self):
             self.plan.provision.requires() +
```

Comment:
We should probably also remove the `provision.requires()` part as it does not make sense now and causes enabling an extra `prepare` step which is not needed anymore. Or just modify the `Provision.requires()` method to return an empty list for now as there could possibly be other requires in the future?

Reply:
hmm, not sure I follow @psss, it now returns an empty list?

Reply:
ah found it, I made it return `[]`